### PR TITLE
Watchdog

### LINKED ISSUE #7578: Move Coda-network Services to Watchdog

### PR DESCRIPTION
**Pull request by es92**
Adds a watchdog helm chart and pod to testnets. Closes #7578

This PR renames coda-network-services to watchdog and moves it to its own pod

After its approved, I'm planning on following it up with a PR changing the report functionality to be cleaned up / more modular and include sending info out to prometheus

Deployed the watchdog-test network to test

### CODE REVIEW

File: automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (0x0I) [inline automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json:1]: Make sense to check these in?

File: helm/watchdog/templates/watchdog.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            value: {{ .Values.makeReportEveryMins | quote }}
+          - name: "MAKE_REPORT_DISCORD_WEBHOOK_URL"
+            value: {{ .Values.makeReportDiscordWebhookUrl | quote }}
+          - name: "MAKE_REPORT_ACCOUNTS"
+            value: {{ .Values.makeReportAccounts | quote }}
+        ports: 
+        - name: metrics-port
+          containerPort: 8000
+        imagePullPolicy: Always
+---
```
Comment (0x0I) [inline helm/watchdog/templates/watchdog.yaml:50]: nit: remove?

**Review by 0x0I — APPROVED** [review]

File: helm/watchdog/templates/watchdog.yaml (hunk 2 above)
Comment (es92) [reply helm/watchdog/templates/watchdog.yaml:50]: The ---?

File: automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json (hunk 1 above)
Comment (es92) [reply automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json:1]: What should I do usually for testnets? For some I think we've said check in / have had to check in (like testworld) , but in general should I not be?

**Review by lk86 — APPROVED** [review]

File: automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json (hunk 1 above)
Comment (lk86) [reply automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json:1]: It's helpful to check all of the files in for debuggability and transparency, so I don't mind this as long as you intend to keep using this testnet as you develop the watchdog. But once this testnet outlives its utility and gets shutdown, it should be removed from the repo as well.

Comment (es92) [reply automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json:1]: Sounds good, makes sense to me (and yep that's the idea)

**Pull request merged**